src/winstanley/Wdl.flex

```diff
@@ -27,7 +27,12 @@ BOOLEAN="true"|"false"
 NUMBER="0"|"1"|"2"|"3"|"4"|"5"|"6"|"7"|"8"|"9"
 
 WHITE_SPACE= \n|\r|\r\n|\ |\t|\f
-TYPE="Array"|"Map"|"Object"|"Boolean"|"Int"|"Float"|"Uri"|"File"|"String"
+
+PRIMITIVE_TYPE="Object"|"Boolean"|"Int"|"Float"|"Uri"|"File"|"String"
```

Comment:
wdl4s considers File to be a primitive.

It was actually news to @cjllanwarne and me that there was a Uri type at all.   Cromwell doesn't support this, it would be nice if Winstanley didn't encourage its use.  😄